README.md

```diff
@@ -61,6 +61,51 @@ schedule.on_calls(date)
 #=> ["someone@example.com", "someone-else@example.com"]
 ```
 
+You can also fetch a timeline for a schedule:
+
+```ruby
+schedule.timeline
+```
+
+Or a specific rotation:
+
+```ruby
+schedule.rotation[0].timeline
+```
+
+You can also specify where you want a timeline to start from:
+
+```ruby
+schedule.timeline(date: Date.parse("2019-01-01"))
+#=> [
+#     {
+#       "id"=>"69e7d46d-538e-4fca-95d0-5c316a54424e",
+#       "name"=>"On Call Phone",
+#       "order"=>2.0,
+#       "periods"=> [...]
+#     },
+#     ...
+#  ]
+```
+
+As well as the interval you want to see a timeline for:
+
+```ruby
+schedule.timeline(interval: 1, interval_unit: :months)
```

Comment:
ISO8601 or GTFO: `P1M`

(joking, this is fine)

Reply:
Might be handy to add a list of valid units though, as this just goes straight into the opsgenie API.

Reply:
I could be super-anal about it and accept an ISO8601 duration and turn it into an Opsgenie-compliant duration 😆